install_config/persistent_storage/topics/glusterfs_example_infra.adoc

```diff
@@ -0,0 +1,97 @@
+. In your inventory file, set the following variables under `[OSEv3:vars]`:
++
+----
+[OSEv3:vars]
+...
+openshift_storage_glusterfs_registry_block_deploy=true
+openshift_storage_glusterfs_registry_block_host_vol_size=50
+openshift_storage_glusterfs_registry_block_storageclass=true
+openshift_storage_glusterfs_registry_block_storageclass_default=true
+
+openshift_storageclass_default=false
+----
++
+[NOTE]
+====
+See the link:{gluster-role-link}[GlusterFS role README] for details on these
+and other variables.
+====
+
+. Add `glusterfs_registry` in the `[OSEv3:children]`
+section to enable the `[glusterfs_registry]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs_registry
+----
+
+. Add a `[glusterfs_registry]` section with entries for each storage node that
+will host the GlusterFS storage. For each node, set `glusterfs_devices` to a
+list of raw block devices that will be completely managed as part of a
+GlusterFS cluster. There must be at least one device listed. Each device must
+be bare, with no partitions or LVM PVs. Specifying the variable takes the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs_registry]
+node11.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node12.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+node13.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
+
+. Add the hosts listed under `[glusterfs_registry]` to the `[nodes]` group:
++
+----
+[nodes]
+...
+node11.example.com openshift_schedulable=True
```

Comment:
Should this be added to the end of each of the 3 lines - openshift_node_labels="{'role': 'infra'}"